src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -17,8 +19,22 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
 
     @Override
     public boolean test(ReadOnlyPerson person) {
-        return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword));
+        Boolean isSelected = keywords.stream().anyMatch(keyword ->
+                StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword));
+        if (isSelected == true) {
+            return isSelected;
+        }
+        for (String keyword: keywords) {
+            if (keyword.length() >= 2 && keyword.substring(0, 2).equals("t/")) {
```

Comment:
Perhaps you can refactor the if statements found in this portion to improve readability of the code.